src/alexapi/shared.py

```diff
@@ -0,0 +1,37 @@
+import yaml
```

Comment:
Get rid of this file. 
- `bcolors` should be in a separate file - [like this](https://github.com/renekliment/AlexaPiNG/blob/816f46d5901637da8ed1689b755b4f133b69aeda/src/alexapi/bcolors.py) - it is unfortunately not merged yet, but will, be, you can use that single file
- from `main.py`, pass the `config` via constructors - this is gonna be for example used when reading audio output device from config
- don't use global `debug` variable in the module - use `self.config['debug']`in the classes or something like that
- in `main.py` before passing `config` in the module (like [here](https://github.com/renekliment/AlexaPiNG/blob/816f46d5901637da8ed1689b755b4f133b69aeda/src/main.py), it's unfortunately not merged yet): 
```
if 'debug' not in config:
	config['debug'] = debug
``` 
- when you've got `debug` like this you don't need the command line parsing stuff
- put `import RPi.GPIO as GPIO` in `player.py` until #30 gets merged ... it's silly, but that's all we can do for now
- whoa, file gone